exporter/opentelemetry-exporter-otlp-proto-grpc/pyproject.toml

```diff
@@ -24,8 +24,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-  "backoff >= 1.10.0, < 2.0.0; python_version<'3.7'",
-  "backoff >= 1.10.0, < 3.0.0; python_version>='3.7'",
+  "backoff >= 1.10.0, < 3.0.0",
```

Comment:
See https://github.com/open-telemetry/opentelemetry-python/issues/2829#issuecomment-1240967454. Are we ok with just doing `~= 2.0`? 

It is hard from a CI perspective for us to test against 1.X and 2.X going forward.

Reply:
Looks like everyone is fine with doing `~= 2.0`. @Yamakaky You can update this.

Reply:
Done